backend/canisters/user_index/impl/src/lifecycle/post_upgrade.rs

```diff
@@ -22,6 +23,33 @@ fn post_upgrade(args: Args) {
 
     canister_logger::init_with_logs(data.test_mode, errors, logs, traces);
 
+    mutate_state(|state| {
```

Comment:
I tend to put a TODO comment to remove this after it is relesaed but doesn't really matter as long as you remember